WalletWasabi/BitcoinP2p/P2pBehavior.cs

```diff
@@ -76,8 +76,10 @@ private async Task ProcessInventoryAsync(Node node, InvPayload invPayload)
 		{
 			if (ProcessInventoryVector(inv, node.RemoteSocketEndpoint))
 			{
-				var newInv = new InventoryVector(InventoryType.MSG_WTX, inv.Hash);
-				getDataPayload.Inventory.Add(newInv);
+				if (inv.Type is InventoryType.MSG_TX or InventoryType.MSG_WTX or InventoryType.MSG_BLOCK)
+				{
+					getDataPayload.Inventory.Add(new InventoryVector(node.AddSupportedOptions(inv.Type), inv.Hash));
```

Comment:
I'm not sure what this does. Should I take it over to https://github.com/zkSNACKs/WalletWasabi/pull/11139?

Reply:
I may understand it now: https://github.com/zkSNACKs/WalletWasabi/pull/11135/files#r1277418655